### ISSUE TITLE
Redis URL logged with credentials

### ISSUE THREAD

**Issue by sidmitra** [launchdarkly/python-server-sdk#210]
**Is this a support request?**
No

**Describe the bug**
Line 24 on [_RedisFeatureStoreCore](https://github.com/launchdarkly/python-server-sdk/blob/a2b221ea5320c0c6e04b0b269315bb0c52ce20eb/ldclient/impl/integrations/redis/redis_feature_store.py#L24) logs the entire redis url to the log under INFO. This could potentially cause redis credentials to be unknowingly logged to centralized logging systems(as was the case with us)

```python
log.info("Started RedisFeatureStore connected to URL: " + url + " using prefix: " + self._prefix)
````

**To reproduce**
Simply enable redis feature store.

**Expected behavior**
One of the following
The redis URL 
- should not be logged
- logged under log.debug
- the username/password should be redacted before logging.

**Logs**
  `Started RedisFeatureStore connected to URL: rediss://____:______@ecx-x-xxx-xxx-xxx.compute-1.amazonaws.com:xxxx using prefix: launchdarkly`

**SDK version**
8.1.3

**Language version, developer tools**
Python 3.11

**OS/platform**
Ubuntu 22

**Comment by keelerm84** [comment]
Thank you for bringing this to our attention. I will update you here once we have a resolution for you.

**Comment by keelerm84** [comment]
Fixed in 8.1.4
